source/releasenotes/about.rst

```diff
@@ -17,85 +17,67 @@
+• API-Driven Console Access
+• Console Access Security Improvements
+• New Global settings UI
+• Configurable MTU for VR
+• Adaptative Affinity Groups
+• Custom DNS Servers for Networks
+• Improved Guest OS Support Framework
+• Support for Enterprise Linux 9
+• Networker Backup Plugin for KVM Hypervisor
+• Custom Quota Tariffs
+• Secure VNC for KVM
 
 The full list of new features can be found in the project release notes at
-https://docs.cloudstack.apache.org/en/4.17.0.0/releasenotes/changes.html
-
-Apache CloudStack Advisory on KVM Shared Mount Point issues on version 4.17.0.0
-===============================================================================
-
-On 14th June 2022, a new issue affecting only KVM users using Shared
-Mount Point storage was reported [1]. This issue affects the creation
-and the usage of existing Shared Mount Point storage pools on Apache
-CloudStack 4.17.0.0.
+https://docs.cloudstack.apache.org/en/4.18.0.0/releasenotes/changes.html
 
-Apache CloudStack 4.17.0.0 added support for the StorPool storage
-based on Shared Mount Point. However, the current version of
-CloudStack doesn't allow multiple implementations of Shared Mount
-Point storage pool providers, causing the StorPool provider to
-override the default implementation. This affected the other storage
-pool providers for Shared Mount Point since CloudStack tries to add
-them as a StorPool storage pool.
+.. _guestosids
 
-To mitigate the issue, a CloudStack administrator needs to do the
-following on version 4.17.0.0:
+Possible Issue with Guest OS IDs
+================================
 
-• On each management server: stop the CloudStack management service, remove the Storpool plugin jar on /usr/share/cloudstack-management/lib/cloud-plugin-storage-volume-storpool-4.17.0.0.jar and restart the Cloudstack management service
-• On each KVM host: stop the CloudStack agent service, remove the StorPool plugin jar on /usr/share/cloudstack-agent/lib/cloud-plugin-storage-volume-storpool-4.17.0.0.jar and restart the CloudStack agent service
+It has been noticed during upgrade testing that some environment, where
+ custom guest OSses where added and mapping for those OSses where added,
+ problems may occur during upgrade. Part of the mitigation is to make sure
+ OSses that are newly mapped but should have already been in the guest_os
+ table are there. Make sure you apply thois before you start the new 4.18
+ management server.
 
-Note: This workaround removes the StorPool plugin support. StorPool
-users should not apply the workaround to continue using their Storpool
-storage.
+first check which you miss:
 
-This issue will be fixed in the upcoming CloudStack version 4.17.1.0.
+.. parsed-literal::
 
-[1] https://github.com/apache/cloudstack/issues/6455
+  SELECT * FROM cloud.guest_os WHERE display_name IN (´CentOS 8´, ´Debian GNU/Linux 10 (32-bit)´, ´Debian GNU/Linux 10 (64-bit)´, ´SUSE Linux Enterprise Server 15 (64-bit)´, ´Windows Server 2019 (64-bit)´)
```

Comment:
I think we've discuss to make this explicitly as an EXAMPLE - i.e. to state here that below things are given just for example purposes. The mitigation makes me believe that anyone having an issue with upgrade should be able to fix it with some/all of the SQLs below - and this is not true - they might be missing other guest OS-es, not only those 4-5.

Reply:
true, but only those five are used in creating new mappings, others are either created in this upgrade and than mapped, or not used in new mappings at all.

Reply:
I can word it more weakly, though.